metrics/opentelemetry-metrics/src/main/scala/sttp/tapir/server/metrics/opentelemetry/OpenTelemetryMetrics.scala

```diff
@@ -80,7 +80,7 @@ object OpenTelemetryMetrics {
   def requestActive[F[_]](meter: Meter, labels: MetricLabels): Metric[F, LongUpDownCounter] =
     Metric[F, LongUpDownCounter](
       meter
-        .upDownCounterBuilder("request_active")
+        .upDownCounterBuilder("http.server.active_requests")
```

Comment:
are all metrics that we should report, included here, or is there anything missing?

Reply:
> are all metrics that we should report, included here, or is there anything missing?

I guess that having those optional metrics ([http.server.request.body.size](https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#metric-httpserverrequestbodysize), 
[http.server.response.body.size](https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#metric-httpserverresponsebodysize)) would be useful :grin:

Reply:
We need support for the content-length header for the above two optional attributes. It seems no content-length header in ServerInterpreter?

Reply:
yeah content length is not that easy to determine. Let's skip it for now

Reply:
I just played yesterday with it but no results